diff -urN gkrellweather-2.0.2.orig/Makefile gkrellweather-2.0.2/Makefile
--- gkrellweather-2.0.2.orig/Makefile	Wed Oct 30 13:47:36 2002
+++ gkrellweather-2.0.2/Makefile	Wed Oct 30 13:53:34 2002
@@ -8,6 +8,9 @@
 
 OBJS = gkrellweather.o
 
+BINDIR=/usr/X11R6/bin
+LIBDIR=/usr/X11R6/lib/gkrellm2/
+
 gkrellweather.so: $(OBJS)
 	$(CC) $(OBJS) -DVERSION=\"$(VERSION)\" -o gkrellweather.so $(LFLAGS) $(LIBS)  -lpthread
 
@@ -18,14 +21,8 @@
 	$(CC) $(CFLAGS) -DVERSION=\"$(VERSION)\" -c gkrellweather.c
 
 install:
-	if [ -d /usr/lib/gkrellm2/plugins/ ] ; then \
-		install -c -s -m 644 gkrellweather.so /usr/lib/gkrellm2/plugins/ ; \
-	elif [ -d /usr/local/lib/gkrellm2/plugins/ ] ; then \
-		install -c -s -m 644 gkrellweather.so /usr/local/lib/gkrellm2/plugins/ ; \
-	else \
-		install -D -c -s -m 644 gkrellweather.so /usr/local/lib/gkrellm2/plugins/gkrellweather.so ; \
-	fi
-	install -c -m 755 GrabWeather /usr/local/bin
+	install -D -c -s -m 644 gkrellweather.so $(DESTDIR)$(LIBDIR)/gkrellweather.so ; \
+	install -c -m 755 GrabWeather $(DESTDIR)$(BINDIR)
 
 uninstall:
 	rm -f /usr/lib/gkrellm2/plugins/gkrellweather.so
