diff -ur gkrellweather-2.0.6.orig/gkrellweather.c gkrellweather-2.0.6/gkrellweather.c
--- gkrellweather-2.0.6.orig/gkrellweather.c	Mon Mar 31 11:12:57 2003
+++ gkrellweather-2.0.6/gkrellweather.c	Sat May 10 20:40:27 2003
@@ -720,7 +720,7 @@
         options.station[1] = c[1];
         options.station[2] = c[2];
         options.station[3] = c[3];
-        sprintf(options.command, PREFIX "/bin/GrabWeather %s", options.station);
+        sprintf(options.command, "GrabWeather %s", options.station);
         sprintf(options.filename, "%s/.wmWeatherReports/%s.dat",
                 getenv("HOME"), options.station);
 	net_update = FALSE;
@@ -983,7 +983,7 @@
     options.windspeed_unit = 1;
     options.pressure_unit = 1;
     strcpy(options.station, DEFAULT_STATION_ID);
-    sprintf(options.command, "/usr/share/gkrellm/GrabWeather %s", options.station);
+    sprintf(options.command, "GrabWeather %s", options.station);
     sprintf(options.filename, "%s/.wmWeatherReports/%s.dat",
             getenv("HOME"), options.station);
 }
